src/components/ens-profile/ProfileSheetHeader.tsx

```diff
@@ -82,20 +89,20 @@ export default function ProfileSheetHeader({
     const isNFTAvatar = avatar && isENSNFTRecord(avatar);
     const avatarUniqueToken = isNFTAvatar && getUniqueToken(avatar);
 
-    const onPressAvatar = () => {
-      if (!avatar || !isNFTAvatar || !avatarUniqueToken) return null;
-      handleSelectNFT(avatarUniqueToken);
-    };
+    let onPressAvatar;
+    if (avatar && isNFTAvatar && avatarUniqueToken) {
+      onPressAvatar = () => handleSelectNFT(avatarUniqueToken);
+    }
```

Comment:
not sure about re-assigning callback fns as a pattern. what was wrong about the other way?

Reply:
i cleaned it a bit, i did it because the onPress function is also a flag to render a pressable button in `ImagePreviewOverlay` https://github.com/rainbow-me/rainbow/pull/3278/files#diff-370bc0ca63de60a719501176e000d7449e78faa68360736718cf79ca22c71211R536